vignettes/complete-multiverse-analysis.Rmd

```diff
@@ -232,32 +233,137 @@ Since the multiverse object has already been created and the one parameter has a
+In our multiverse we have identified 5 options for calculating `fertility`, 3 options for calculating `menstrual calculation` and `relationship status` each, 3 wyas of excluding participants based on their `cycle length` and 2 ways of excluding participants based on the self-reported `certainty` of their responses.
+
+This results in $ 5 \times 3 \times 3 \times 3 \times 2 = 270$ possible combinations.
 
 ```{r}
-M$df
+multiverse_table(M) %>% nrow()
 ```
 
-<span style="colour:"red"> __this does not work__ </span>
+We can then inspect the default analysis the default single universe analysis from this multiverse:
 
-Or, we could get the code for a different single universe analysis by providing a list of parameter assignments (options which each parameter should take) to the `execute_default()` function and execute it:
+```{r}
+M$df %>%
+  head()
+```
+
+```{r, eval = FALSE, include = FALSE}
+#Or, we could get the code for a different single universe analysis by providing a list of parameter assignments (options which each parameter should take) to the `execute_default()` function and execute it:
 
-```{r, eval = FALSE}
 universe_parameter_assignment <- list(
     menstrual_calculation = "mc_option",
     relationship_status = "rs_option3",
     cycle_length = "cl_option2",
     certainty = "cer_option1",
     fertile = "fer_option4"
   )
+
 M %>% execute_default( universe_parameter_assignment ) %>%
   select(ComputedCycleLength, NextMenstrualOnset, CycleDay, Relationship, Fertility, everything()) %>%
   head()
 ```
+
+
+## Specifying conditions in the multiverse analysis
+In a multiverse analysis, it may occur that the value of one variable might depend on the value of another variable defined previously. 
+For example, in our example, we are excluding participants based on their *cycle length*. This can be done in two ways: we can use the values of the variable,`ComputedCycleLength` or `ReportedCycleLength`. If we are using `ComputedCycleLength` to exclude participants, this means that we should not calculate the variable `NextMenstrualOnset` (date for the onset of the next menstrual cycle) using the `ReportedCycleLength` value. Similarly, if we are using `ReportedCycleLength` to exclude participants it is inconsistent to calculate `NextMenstrualOnset` using `ComputedCycleLength`.
+
+We should be able to express these conditions in the multiverse. We can do this in two ways:
+1. `%when%`: when declaring a branch, we can use this operator to specify the condition as `A %when% B`
+2. `branch_assert`: the conditional $A | B$ is also referred to as the connective $A \implies B$. This has the meaning "if A is true, then B is also true" and is an abbreviation for $\neg A | B$
+
+#### The %when% operator
+There are two ways in which you can specify the `%when%` operator. The first is to specify it at the end of the branch. This will work even if you omit the branch option name.
```

Comment:
Makes sense --- given that, it would be good to have an example of that usage (with no option name)

Reply:
Should this be in a different example then? I'd probably err on the side of omitting the last sentence because in this example we don't mention options without branch option names

Reply:
Or if there's an example of unnamed branches elsewhere in the docs you could point people to it, and then modify the unnamed branch docs to make sure there's an example of a conditional in them